internal/pkg/acl/acl.go

```diff
@@ -205,6 +205,18 @@ func convertToFlags(operations ...any) string {
 	return strings.Join(ops, ", ")
 }
 
+func ConvertToLower(operations ...any) string {
+	var ops []string
+
+	for _, v := range operations {
+		s := strings.ToLower(strings.ReplaceAll(fmt.Sprint(v), "_", "-"))
```

Comment:
You could move `strings.ToLower()` outside of the loop.

Reply:
I moved it out of `ConvertToLower` but not the `convertToFlags` functions because it messes up the sorting (unless I change the hard-coded `consumer-group` and `cluster-scope` strings to upper case, which I can do if we think that's better).

Reply:
How does it mess up the sorting? Do we want something other than alphabetical order?

Reply:
Moving `ToLower` outside results in `cluster-scope` and `consumer-group` in lowercase because of the special handling and `TOPIC` and `TRANSACTIONAL-ID` for the other two. The sorting puts the capitalized entries first.

Reply:
Not 100% sure I understand, but let's do whatever it takes to alphabetize these.